tools/deb/istio-iptables.sh

```diff
@@ -0,0 +1,111 @@
+#!/bin/bash
```

Comment:
The k8s iptable rules have a corresponding test (see https://github.com/istio/pilot/blob/master/docker/prepare_proxy_test.sh). It would be nice to see something similar for the VM rules. This will give us more confidence when we go to merge the VM and k8s rules.

To address @rshriram's comment, one way to make this reviewer easier is to commit a copy of the prepare_proxy.sh first and rebase these changes on top of that. The PR would then include the diff vs. the version we're already familiar with and highlight how far the VM script has diverged from k8s script.